GameInterface.Tests/Serialization/SerializerTests/WeaponDesignSerializationTest.cs

```diff
@@ -63,7 +63,7 @@ public void WeaponDesign_Full_Serialization()
             };
 
             CraftingTemplate craftingTemplate = MBObjectManager.Instance.CreateObject<CraftingTemplate>();
-            typeof(CraftingTemplate).GetField("_buildOrders", BindingFlags.Instance | BindingFlags.NonPublic).SetValue(craftingTemplate, buildOrders);
+            typeof(CraftingTemplate).GetProperty("BuildOrders", BindingFlags.Instance | BindingFlags.Public).SetValue(craftingTemplate, buildOrders);
```

Comment:
The default flags are `BindingFlags.Instance | BindingFlags.Public` so those are not needed
Also use `nameof` here so we get a compiler error instead of a runtime error
Make this into a `private static readonly` field and use that here
```suggestion
            typeof(CraftingTemplate).GetProperty(nameof(CraftingTemplate.BuildOrders)).SetValue(craftingTemplate, buildOrders);
```